assets/html/documenter.css

```diff
@@ -560,7 +564,7 @@ article section.docstring a.source-link {
     }
 
     article > header div#topbar span {
-        position: fixed;
+        position: relative;
```

Comment:
The `position: fixed` has always seemed like a bug to me on mobile.  I think this is the correct fix to make it properly hide along with the menu when scrolling.